operator/src/main/java/oracle/kubernetes/operator/calls/RequestParams.java

```diff
@@ -46,4 +51,31 @@ public RequestParams(
   public String getLabelSelector() {
     return callParams.getLabelSelector();
   }
+
+  public String toString() {
+    return toString(true);
+  }
+
+  /**
+   * Create readable details of request, optionally including the request body.
+   * @param includeBody true if request body should be included
+   * @return readable details of the request
+   */
+  public String toString(boolean includeBody) {
+    StringBuilder sb = new StringBuilder();
+    sb.append(call);
+    if (namespace != null) {
+      sb.append(' ');
+      sb.append(LOGGER.formatMessage(MessageKeys.REQUEST_PARAMS_IN_NS, namespace));
+    }
+    if (name != null) {
+      sb.append(namespace != null ? ", " : " ");
+      sb.append(LOGGER.formatMessage(MessageKeys.REQUEST_PARAMS_FOR_NAME, name));
+    }
+    if (includeBody && body != null) {
+      sb.append(namespace != null || name != null ? ", " : " ");
```

Comment:
what does this mean? which is evaluated first the or, or the elvis?
is that what you intended? 
seems like some parens here would help future generations greatly